test/e2e/test-errortracker.js

```diff
@@ -74,7 +74,7 @@ describe('Error Tracker Server', () => {
   const referrer = 'https://cdn.ampproject.org/';
   const userAgent = 'Google Chrome blah blah version';
   const knownGoodQuery = Object.freeze({
-    version: '011830043289240',
```

Comment:
Revert these changes, I purposefully made them `011830043289240` because it's 10 years away.

Reply:
Done.